google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java

```diff
@@ -829,6 +861,22 @@ public Builder setEndpoint(String endpoint) {
       return this;
     }
 
+    /** Gives the ability to enable gRPC compression. */
+    public Builder setEnableCompression(boolean enableCompression) {
+      this.enableCompression = enableCompression;
+      return this;
+    }
+
+    /**
+     * Gives the ability to set the threshold in bytes above which gRPC compression happens. But to
```

Comment:
"Sets the threshold (in bytes) above which messages are compressed for transport. Only takes effect if `setEnableCompression(true)` is also called."